data.sql

```diff
@@ -10,6 +11,26 @@ VALUES ('Pikachu', '2021-1-7', 1, FALSE,15.04);
 
 INSERT INTO animals(name, date_of_birth, escape_attempts, neutered,weight_kg)
 VALUES ('Devimon', '2017-5-12', 5, TRUE,11.00);
--- INSERT INTO animals (name) VALUES ('Luna');
--- INSERT INTO animals (name) VALUES ('Daisy');
--- INSERT INTO animals (name) VALUES ('Charlie');
+
+
+--Week1 Project2
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered , weight_kg )
+VALUES ('Charmander', '2020-2-8', 0, FALSE ,11);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered , weight_kg )
+VALUES ('Plantmon', '2020-11-15', 2, TRUE ,5.7);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered , weight_kg )
+VALUES ('Squirtle', '1993-4-2', 3, FALSE ,12.13);
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered , weight_kg )
+VALUES ('Angemon', '2005-6-12', 1, TRUE ,45);
```

Comment:
- [ ] According to the project [requirements](https://github.com/microverseinc/curriculum-databases/blob/main/db-structure/vet_clinic_query_update_tables.md), these values of weight are negative, please kindly fix that.